packages/kbn-handlebars/src/upstream/index.compiler.test.ts

```diff
@@ -71,6 +67,7 @@ describe('compiler', () => {
 
       it('should not modify the options.data property(GH-1327)', () => {
         const options = { data: [{ a: 'foo' }, { a: 'bar' }] };
+        // @ts-expect-error: The `data` property is supposed to be a boolean, but in this test we want to ignore that
         compile('{{#each data}}{{@index}}:{{a}} {{/each}}', options)({});
```

Comment:
__optional nit:__ wondering if it'd be safer to use `as unknown as boolean` instead of `ts-expect-error` to not miss any other unrelated type errors (e.g. different number of args for compile function or the function it returns)?
```suggestion
        // The `data` property is supposed to be a boolean, but in this test we want to ignore that
        const options = { data: [{ a: 'foo' }, { a: 'bar' }] as unknown as boolean };
        compile('{{#each data}}{{@index}}:{{a}} {{/each}}', options)({});
```

Reply:
Good point 👍